charts/argo-rollouts/README.md

```diff
@@ -94,6 +94,7 @@ For full list of changes please check ArtifactHub [changelog].
 | controller.metrics.service.annotations | object | `{}` | Service annotations |
 | controller.metrics.service.port | int | `8090` | Metrics service port |
 | controller.metrics.service.portName | string | `"metrics"` | Metrics service port name |
+| controller.metrics.serviceMonitor.namespace | string | `""` | Namespace to be used for the ServiceMonitor |
```

Comment:
alphabetical.  needs to be under `controller.metrics.serviceMonitor.metricRelabelings`